platform-coop-toolkit/coop-library-framework
# Add proper readme and utility script (resolve #107)

## Description

Adds a readme. Changelogs and updates should be added to readme.txt, and Markdown files (README.md, CHANGELOG.md and FAQ.md) can be generated using `npm run readme`.

## Steps to test

1. Edit readme, changelog, or FAQ in readme.txt.
2. Run `npm run readme`.

**Expected behavior:** Corresponding Markdown files should be updated.

## Additional information

- https://www.npmjs.com/package/gulp-readme-to-markdown

## Related issues

- Resolves #107
